docs/src/pages/components/authenticator/index.page.mdx

```diff
@@ -449,6 +449,21 @@ These [translations](https://github.com/aws-amplify/amplify-ui/blob/main/package
 
 The list of available keys are available [here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts).
 
+#### Confirm Sign Up Page Translations
+
+The confirm sign up page has a few specialized strings that can be translated.
+These include:
+
+```
+Your code is on the way. To log in, enter the code we emailed to
+
+Your code is on the way. To log in, enter the code we texted to
+
+Your code is on the way. To log in, enter the code we sent you. It may take a minute to arrive.
+
+. It may take a minute to arrive.
+```
```

Comment:
Added new section to docs on these new translations, since they are not obvious.